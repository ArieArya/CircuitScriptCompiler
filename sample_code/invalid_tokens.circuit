wire invalid-id;
reg invalid_op = !0;
reg invalid_digit = 2;
