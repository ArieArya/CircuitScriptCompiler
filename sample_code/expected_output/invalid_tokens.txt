Lexing file: invalid_tokens.circuit
Tokens:
Token.KEYWORD (Lexeme = 'wire')
Token.IDENTIFIER (Lexeme = 'invalid')
Token.IDENTIFIER (Lexeme = 'id')
Token.SEMICOLON (Lexeme = ';')
Token.KEYWORD (Lexeme = 'reg')
Token.IDENTIFIER (Lexeme = 'invalid_op')
Token.OPERATOR (Lexeme = '=')
Token.DIGIT (Lexeme = '0')
Token.SEMICOLON (Lexeme = ';')
Token.KEYWORD (Lexeme = 'reg')
Token.IDENTIFIER (Lexeme = 'invalid_digit')
Token.OPERATOR (Lexeme = '=')
Token.SEMICOLON (Lexeme = ';')

Errors:
Error parsing '-' at index 12.
Error parsing '!' at index 34.
Error parsing '2' at index 58.
